src/components/Button/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+interface ButtonProps {
+    type?: 'button' | 'submit' | 'reset';
+    className: string;
+    onClick: (event: React.MouseEvent<HTMLButtonElement>) => void;
+    children: string;
+}
+
+const Button: React.FC<ButtonProps> = ({ type, className, onClick, children }) => {
```

Comment:
Nit: I think you have to pass disabled prop in Button component
correct me if i am wrong

Reply:
Depends on the use case. Here there is no need to disable it as of now.

Reply:
You made reusable component...Right??